man/cytoCytoscape.Rd

```diff
@@ -27,7 +27,7 @@ class attribute).}
 
 \item{style}{Named list. Add inline styles to the root element.}
 
-\item{elements}{List. A list of dictionaries representing the elements of the networks.
+\item{elements}{List of named lists. A list of dictionaries representing the elements of the networks.
```

Comment:
```suggestion
\item{elements}{List of named lists ('dictionaries'). A list of dictionaries representing the elements of the networks.
```

Since the rest of the description uses the word "dictionary", it might be a good idea to introduce it here!

Reply:
The issue here is actually the preceding description containing the type, since dictionaries don't exist in R. 

I'm not quite sure how to handle this, although I'm starting to think a regex to search-and-replace `dictionary|dictionaries|dict` within the description fields of the R help may not be a bad idea.

@alexcjohnson Any thoughts on whether it's remotely safe to automate this replacement for generated R packages?

Reply:
Whatever we do here it needs to happen in a generic way in the component generator, not by hand in each package. Honestly we're in a weird situation already, where we have docstrings in the javascript source that refer to python data types - there's no "dictionary" in javascript either. Short-term since the main offender is `[Dd]ict(ionar(y|ies)|s)?` we could just replace that with a regex in the R docs. Long term I'm starting to think we should use js data types in the source code but somehow label them so we can do an efficient and robust replacement in each target language.

Reply:
👍 to the idea of using JS data types in the source code. This feels philosophically compatible with a recent conversation @Marc-Andre-Rivet and I had about JS serving as a "neutral" third language. 

It feels logical that the JS docstrings only document JS features, and that all other languages would substitute in the appropriate terms as needed.